internal/ux/messages.go

```diff
@@ -10,17 +10,22 @@ import (
 	"github.com/fatih/color"
 )
 
-func Fsuccess(w io.Writer, format string, a ...any) {
+func FSuccess(w io.Writer, format string, a ...any) {
 	color.New(color.FgHiGreen).Fprint(w, "Success: ")
 	fmt.Fprintf(w, format, a...)
 }
 
-func Fwarning(w io.Writer, format string, a ...any) {
+func FWarning(w io.Writer, format string, a ...any) {
 	color.New(color.FgHiYellow).Fprint(w, "Warning: ")
 	fmt.Fprintf(w, format, a...)
 }
 
-func Ferror(w io.Writer, format string, a ...any) {
+func FError(w io.Writer, format string, a ...any) {
 	color.New(color.FgHiRed).Fprint(w, "Error: ")
 	fmt.Fprintf(w, format, a...)
 }
+
+func FNotice(w io.Writer, format string, a ...any) {
+	// TODO: Add a notice prefix?
+	color.New(color.FgYellow).Fprintf(w, format, a...)
```

Comment:
🤔 Yeah, I think we should change this to have a prefix It'll fit the convention of the rest of our messages.

Thinking a bit more, I have a slight preference for changing "notice" to "info" since its easier to read at a quick glance. 

Could we rename this to `Finfo` (lowercase i) and have a yellow "Info: " prefix?

Reply:
Done. PTAL